### ISSUE TITLE
No packages available

### ISSUE THREAD

**Issue by spuder** [Appdynamics/appdynamics-cookbooks#28]
It appears that the default URL provided by app dynamics is invalid. 

>https://packages.appdynamics.com/dotnet/latest/dotNetAgentSetup64.msi

I'm unable to find what the correct url would be. 

This results in the following error. 

```bash
================================================================================
             Error executing action `create` on resource 'remote_file[C:\Users\vagrant\AppData\Local\Temp\kitchen\cache\package\dotNetAgentSetup64.msi]'
             ================================================================================

             Net::HTTPServerException
             ------------------------
             403 "Forbidden"

             Compiled Resource:
             ------------------
             # Declared in

             remote_file("C:\Users\vagrant\AppData\Local\Temp\kitchen\cache\package\dotNetAgentSetup64.msi") do
        provider Chef::Provider::RemoteFile
        action [:create]
        retries 0
        retry_delay 2
        default_guard_interpreter :default
        path "C:\\Users\\vagrant\\AppData\\Local\\Temp\\kitchen\\cache\\package\\dotNetAgentSetup64.msi"
        atomic_update true
        source ["https://packages.appdynamics.com/dotnet/latest/dotNetAgentSetup64.msi"]
        use_etag true
        use_last_modified true
             end
```

**Comment by dkoepke** [comment]
The URL for the .NET package is https://packages.appdynamics.com/dotnet/4.1.2.0/dotNetAgentSetup64-4.1.2.0.msi. We need to drop support for "latest" in the cookbooks and fix the default package sources.

**Comment by dkoepke** [comment]
Oops. Didn't mean to close this since there's work to do on it.

**Comment by spuder** [comment]
Thanks for the response. I changed the kitchen file to use version 4.1.2.0. When I run a kitchen converge, it attempts to download from the following location. 

https://packages.appdynamics.com/dotnet/4.1.2.0/dotNetAgentSetup64.msi

Compare that to 

https://packages.appdynamics.com/dotnet/4.1.2.0/dotNetAgentSetup64-4.1.2.0.msi

Do you have a time frame for when this might be fixed? I don't want to submit a pull request if someone else is presently working on it. 

Basically just one line needs to be changed. (After you remove all the 'latest' references)

    default['appdynamics']['dotnet_agent']['package_file'] = "dotNetAgentSetup64-#{node['appdynamics']['dotnet_agent']['version']}.msi"

This needs more testing because of this: https://coderanger.net/derived-attributes/

**Comment by dkoepke** [comment]
I'll make the change today.

**Comment by spuder** [comment]
I ran a kitchen converge on fix/package-source. No errors.

**Comment by dkoepke** [comment]
Fixed in #30.
